tests/tests/swfs/from_avmplus/e4x/Namespace/e13_2_5/test.toml

```diff
@@ -1,2 +1 @@
 num_ticks = 1
-known_failure = true
```

Comment:
This does not pass yet due to https://github.com/ruffle-rs/ruffle/pull/16393#discussion_r1608150947